PICMI_Python/interactions.py

```diff
@@ -0,0 +1,21 @@
+"""Classes following the PICMI standard
+These should be the base classes for Python implementation of the PICMI standard
+The classes in this file are related to interactions (e.g. field ionization, collisions, QED)
+"""
+
+from .base import _ClassWithInit
+
+class PICMI_FieldIonization(_ClassWithInit):
+    """
+    Field ionization on an ion species
+    - model: ionization model, e.g. "ADK" (string)
```

Comment:
Can you update the doc string to use the numpy style formatting that is used elsewhere in picmi?